webapp/components/Registration/CreateUserAccount.spec.js

```diff
@@ -81,7 +82,7 @@ describe('CreateUserAccount', () => {
               email: 'sixseven@example.org',
               nonce: '666777',
               password: 'hellopassword',
-              termsAndConditionsAgreedVersion: '0.0.2',
+              termsAndConditionsAgreedVersion: VERSION,
```

Comment:
Yesterday in the lecture I told the students, that hard-coded values in the tests are OK :laughing: I think in this very situation it's fine to import the `VERSION` constant